configs/user-service/user-service.dockerfile

```diff
@@ -8,13 +8,14 @@ WORKDIR /service
 
 ENV PYTHONUNBUFFERED=1
 
+# Required for psycog2 build
+RUN apt-get update && apt-get install libpq-dev python3-dev gcc -y
+
```

Comment:
swapped to alpine to reduce fat and add dependencies for gcc + psycorg2, should work on mac as well